zerver/webhooks/jotform/tests.py

```diff
@@ -0,0 +1,29 @@
+from zerver.lib.test_classes import WebhookTestCase
+
+
+class JotFormHookTests(WebhookTestCase):
+    STREAM_NAME = 'test'
+    URL_TEMPLATE = "/api/v1/external/jotform?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = "jotform"
+
+    def test_response(self) -> None:
+        expected_title = "JotForm-Response"
+        expected_message = """
+```
+Form Title: Form
+Form Id: 202944822449057
+Submission Id: 4791133489169827307
+-------------------------
+
+Name : gerd sfds
+Address : asfdsf dsgd asd grgr 165
+Signature : uploads/gauravguitarrocks/202944822449057/4791133489169827307/4791133489169827307_signature_4.png
+```
```

Comment:
This formatting could use some work to be more readable.  E.g.:
* The space
* The form title could go in the topic
* The form/submission IDs could be encoded in a link to the response
* The name/address could be put in one line together:, e.g. "Name <address> submitted a [response](URL)." if "address" is supposed to be an email (if so, the test fixtures are a bit weird).

Reply:
I am not sure about the last two points particularly last one as the user may not have the name field in the form as it is not a required field.

Reply:
The address was physical location address 😄

Reply:
Yeah, OK, if it's physical that doesn't work.  But please remove the space before the `:`, adjust the title, and consider using a bulleted list formatting to make it look nicer.

Reply:
Sure, I will rectify them.